packages/vscode-extension/src/webview/components/ToolsDropdown.tsx

```diff
@@ -96,6 +96,15 @@ function ToolsDropdown({ children, disabled }: { children: React.ReactNode; disa
             <span className="codicon codicon-chip" />
             {isProfilingCPU ? "Stop JS CPU Profiler" : "Start JS CPU Profiler"}
           </DropdownMenu.Item>
+          <DropdownMenu.Item
+            className="dropdown-menu-item"
+            onSelect={() =>
+              isProfilingReact ? project.stopProfilingReact() : project.startProfilingReact()
```

Comment:
What happens if `startProfilingReact()` is called twice, without calling `stopProfilingReact()`?
Can't that happen if the user tries to toggle this quickly, while the extension hasn't sent the `isProfiling` event yet?

Reply:
this is safe to be called, the subsequent calls are going to be ignored.